File: pgm/ABC_Dart/ABC_384_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final List<String> arr = stdin.readLineSync()!.split(' ');
  final String S = stdin.readLineSync()!;
  const int one = 1;
  const int two = 2;
  const int zero = 0;
  final [int _, String c1, String c2] = <dynamic>[
    int.parse(arr[zero]),
    arr[one],
    arr[two],
  ];
  print(S.replaceAll(RegExp('[^${RegExp.escape(c1)}]'), c2));
  exit(0);
}

File: pgm/ABC_Kotlin/ABC_384_A.kt
// import java.time.*

// import kotlin.math.*

const val one: Int = 1
const val two: Int = 2
const val zero: Int = 0
const val space: String = " "

fun main() {
    val arr: List<String> = readln().split(space)
    val (_: Int, c1: String, c2: String) = Triple(arr[zero].toInt(), arr[one], arr[two])
    val S: String = readln()
    println(S.replace(Regex("[^${Regex.escape(c1.toString())}]"), c2))
    kotlin.system.exitProcess(0)
}